x11-libs/qscintilla/qscintilla-2.14.1-r1.ebuild

```diff
@@ -0,0 +1,112 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit flag-o-matic multibuild qmake-utils
```

Comment:
Not that dangerous but:

`UnusedInherits: version 2.14.1-r1: unused eclass: qmake-utils`

Reply:
hmm, this could be a misleading warning :eyes: 
```
			qt5) local QMAKE=eqmake5 ;;
			qt6) local QMAKE=eqmake6 ;;
```
these lines (see L82 in code below) rely on qmake-utils, not? https://github.com/gentoo/gentoo/blob/c09a818d5982566010ed9439502eaa3854187c1f/eclass/qmake-utils.eclass#L103

... or are there better ideas around?

Reply:
@Chiitoo , looks like the QA check tool missed the eqmake5/eqmake6 "hidden" in a variable, but qmake-utils are very necessary indeed.